### ISSUE TITLE
Cannot run on IOS

### ISSUE THREAD

**Issue by mvpdream** [alinz/react-native-webview-bridge#142]
![5443782a-f1c4-43fb-a71c-96f3fc2787d1](https://cloud.githubusercontent.com/assets/16203724/17884872/224eac30-694d-11e6-8d1e-dd3bd3671c42.png)

**Comment by jqn** [comment]
Same problem here

**Comment by santong** [comment]
same problem

**Comment by dangnelson** [comment]
This pull request appears to fix the issue: https://github.com/alinz/react-native-webview-bridge/pull/129/files

**Comment by dangnelson** [comment]
But now I'm getting this error: https://github.com/alinz/react-native-webview-bridge/issues/135

**Comment by fanliming** [comment]
@dangnelson  did you fix #135 problem ? I add a feedback under the issue

**Comment by dangnelson** [comment]
Nope, I just reverted to RN 0.24.0 for now.

**Comment by mibi-b** [comment]
I fixed compile issue by adding to RCTWebViewBridge.m

`#import <objc/runtime.h>`

after the other imports.


```
#import "RCTView.h"
#import "UIView+React.h"

#import <objc/runtime.h>
```

**Comment by turtec** [comment]
#import <objc/runtime.h>

works for me thanks

**Comment by hmedney** [comment]
Fixed in PR #150 and published to npm
